stories/grid/align.js

```diff
@@ -37,56 +38,52 @@ story.add('Vertical Align', () => {
 
   return (
     <WithNotes notes={notes}>
-      <div className="gl-layout-root">
-        <div className="gl-layout-content">
-          <h1>Item Align</h1>
-          <Grid margin={margin} stretch={stretch}>
-            <ReferenceColumn height={height} />
-            <Box size={2} type="B" value="TOP" align={Item.ALIGN.TOP} />
-            <Box size={2} type="C" value="MIDDLE" align={Item.ALIGN.MIDDLE} />
-            <Box size={2} type="D" value="BOTTOM" align={Item.ALIGN.BOTTOM} />
-            <Box size={2} type="C" value="MIDDLE" align={Item.ALIGN.MIDDLE} />
-            <Box size={2} type="E" value="DEFAULT" />
-            <ReferenceColumn height={height} />
+      <Root>
+        <h1>Item Align</h1>
+        <Grid margin={margin} stretch={stretch}>
+          <ReferenceColumn height={height} />
+          <Box size={2} type="B" value="TOP" align={Item.ALIGN.TOP} />
```

Comment:
sure yeah we can do that too, I wanted to be explicit that those came from Item and not something else defined by Box but I can see a case to do it based of Box too since I'm using Box here